2_observations/src/data_munge_functions.R

```diff
@@ -19,9 +26,12 @@ filter_temp_data <- function(cross_ind, dat_ind, ngwos_ind, out_ind) {
   sites <- readRDS(sc_retrieve(cross_ind)) %>%
     select(site_id, subseg_id, seg_id_nat) %>%
     distinct(site_id, subseg_id, seg_id_nat, .keep_all = TRUE)
+
   dat <- readRDS(sc_retrieve(dat_ind))
-  ngwos_dat <- readRDS(sc_retrieve(ngwos_ind))
-  
+  ngwos_dat <- readRDS(sc_retrieve(ngwos_ind)) %>%
+    mutate(site_id = paste0('USGS-', site_id)) %>%
+    rename(temp_degC = temp_c)
```

Comment:
I think I started using all `temp_degC`. Are you using all `temp_c`? I'm fine either way, but it'd be good for me to know, that way I can be consistent.